server/app/controller/question.js

```diff
@@ -1,53 +1,84 @@
-
-
-const Questions = [];
-// Get all Questions
-const getAll = () => Promise.resolve(Questions);
-
-// Get one question
+import db from '../db/dbconnect';
+/** Get all question
+ * @return {obj}
+ * @public
+*/
+const getAll = () => {
+  return db.any('select * from questions')
+    .then((data) => {
+      return Promise.resolve(data);
+    });
+};
+/**  Get one question
+ * @param {string}
+ * @return {obj}
+ * @public
+*/
 const getOne = (id) => {
-  const questionId = parseInt(id, 10);
-  const question = Questions.find(questn => questn.id === questionId);
-  if (!question) return Promise.reject(new Error('pass valid id'));
-  return Promise.resolve(question);
+  const questionId = parseFloat(id);
+  return db.task(t => t.batch([
```

Comment:
Nice work. However, the use of a single alphabet character as variable is discouraged(in this case `t`). For more readability and maintainability, use a more descriptive variable name that describes the values it holds. Please let this reflect everywhere in your code base.